frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-page.tsx

```diff
@@ -97,23 +212,45 @@ export const OperatorHubList: React.FC<OperatorHubListProps> = ({
             [OperatorHubCSVAnnotationKey.supportWorkflow]: marketplaceSupportWorkflow,
           } = currentCSVAnnotations;
 
+          const getSubscription = subscriptionFor(subscription.data)(operatorGroup.data)(
+            pkg.status.packageName,
+          )(namespace);
+
+          const operatorCSV = clusterServiceVersionFor(
+            pkg,
+            clusterServiceVersions,
+            currentCSVDesc?.version,
+          );
+
+          const operatorInstallPlan = installPlanFor(getSubscription, installPlan);
+
+          const installed = isOperatorInstalled(
+            subscription,
```

Comment:
`subscription` is of type `{ data: SubscriptionKind[] }`, but we treat it like a `SubscriptionKind` in the function. I have a feeling there is a bug here.